### DISCUSSION TITLE
GPX Track Download

### DISCUSSION THREAD

**Discussion by Karl60** [streetcomplete/StreetComplete#3950] [Q&A] [answered]
Is there a way to download a gpx track from the app?  I note the app is recording my route - as a nice blue line - so it would be useful if this track could be downloaded as a gpx and/or directly uploaded to OSM as a Public GPS Trace.

**Comment by riQQ** [comment, marked as answer]
As of now, there is no way to download a GPX track from the app.
GPS trace upload to OSM is currently being implemented via #3573.
